lib/search/hotel.js

```diff
@@ -53,7 +53,7 @@ function getTileFilters (tiles, callback) {
   if (tiles && tiles.length) {
     var params = {
       linkType: ':FILTERS',
-      maxDept: 1
+      maxDept: 5
```

Comment:
I would use `maxDepth` (*with the `h` on the end*) here for clarity.
something to consider on next update. 👍